.vale/styles/RedHat/TermsErrors.yml

```diff
@@ -303,3 +303,24 @@ swap:
   wish: want
   wish|would like: want
   zero out: zero
+  byte code: bytecode
+  client: customer
+  computer farm|computer ranch: server cluster
+  details pane: details view
+  domain mode: managed domain
+  enum: data enumeration
+  live broker: master broker
+  live only: live-only
+  load-balance|load-balancing: load balance
+  multi site|multi-site: multisite
+  passive broker: slave broker
```

Comment:
Seems to violate our conscious language guidelines.

Reply:
And yet: https://redhat-documentation.github.io/supplementary-style-guide/#slave-broker

Reply:
Good point. We don't set policy, we encode it.